src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -191,11 +191,11 @@ JSON file. See [Common metadata fields](#common-metadata-fields) for a
+| ContrastBolusIngredient | OPTIONAL              | [string][]                             | Active ingredient of agent. Values MUST be one of: IODINE, GADOLINIUM, CARBON DIOXIDE, BARIUM, XENON Corresponds to DICOM Tag 0018,1048.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| RepetitionTimeExcitation | OPTIONAL             | [number][] or [array][] of [numbers][] | The time in seconds between successive excitation pulses that excite a spin system. A single value applies to all volumes, or an array provides a value for each volume. The DICOM tag that best refers to this parameter is [(0018, 0080)](http://dicomlookup.com/lookup.asp?sw=Tnumber&q=(0018,0080)). This field may be used together with the `RepetitionTimePreparation` for certain use cases, such as [MP2RAGE](https://infoscience.epfl.ch/record/172927/files/mp2rage.pdf). Use `RepetitionTimeExcitation` (in combination with `RepetitionTimePreparation` if needed) for anatomy imaging data rather than `RepetitionTime` as it is already defined as the amount of time that it takes to acquire a single volume in the [task imaging data](#task-including-resting-state-imaging-data) section. 4D files with variable `RepetitionTimeExcitation` should only be used in volume timing is critical for interpretation of the data, as in ASL. |
```

Comment:
```suggestion
| RepetitionTimeExcitation | OPTIONAL             | [number][] | The time elapses (in seconds) between two successive excitations. The DICOM tag that best refers to this parameter is [(0018, 0080)](http://dicomlookup.com/lookup.asp?sw=Tnumber&q=(0018,0080)). This field may be used together with the `RepetitionTimePreparation` for certain use cases, such as [MP2RAGE](https://infoscience.epfl.ch/record/172927/files/mp2rage.pdf). Use `RepetitionTimeExcitation` (in combination with `RepetitionTimePreparation` if needed) for anatomy imaging data rather than `RepetitionTime` as it is already defined as the amount of time that it takes to acquire a single volume in the [task imaging data](#task-including-resting-state-imaging-data) section. |
```

Reply:
After our meeting with @patsycle, we noticed that there is no need for this field to specify requirement level with two options, which is also favourable for the [potential generic use](https://github.com/bids-standard/bids-specification/pull/671#issuecomment-728202971) of this field in BIDS v2.0. I slightly changed the wording in that regard.